### PR TITLE
Use the workspace root to translate error message paths to full paths

### PR DESCRIPTION
**Pull request by rtsuk**

### CODE REVIEW

File: Support/lib/build.rb [hunk 1]
```diff
@@ -29,6 +29,26 @@ def cargo_params
   ENV.key?('TM_CARGO_PARAMS') ? ENV['TM_CARGO_PARAMS'].split(' ') : []
 end
 
+# Try to use cargo metadata to find the workspace
+# root for resolving relative paths in error messages.
+# Fall back to TM_PROJECT_DIRECTORY if that attempt
+# fails for any reason.
+def get_worspace_root(cargo_home)
```
Comment (carols10cents) [inline Support/lib/build.rb]: Hm, is this function called `get_worspace_root` rather than `get_workspace_root` for any particular reason?

**Comment by carols10cents** [comment]
Also there's merge conflicts now that I merged the other PR :)

File: Support/lib/build.rb (hunk 1 above)
Comment (rtsuk) [reply Support/lib/build.rb]: Nope, just a typo.

**Comment by rtsuk** [comment]
I think I fixed the typo but I am reluctant to hit something so pejorative as "Dismiss Review".

**Pull request merged**